operator/config/rbac/role.yaml

```diff
@@ -113,6 +113,7 @@ rules:
   - core.cnrm.cloud.google.com
   resources:
   - configconnectors/status
+  - configconnectors/finalizers
```

Comment:
Is `update` the only required verb for `configconnectors/finalizers`?

If so could you add a separate entry for it?

```
- apiGroups:
  - core.cnrm.cloud.google.com
  resources:
  - configconnectors/finalizers
  verbs:
  - update
 ```

Reply:
ok